lib/routes/scmp/index.js

```diff
@@ -0,0 +1,94 @@
+const parser = require('@/utils/rss-parser');
+const cheerio = require('cheerio');
+const got = require('got');
+
+module.exports = async (ctx) => {
+    const categoryId = ctx.params.category_id;
+    const rssUrl = `https://www.scmp.com/rss/${categoryId}/feed`;
+    const feed = await parser.parseURL(rssUrl);
+    const chromeMobileUserAgent = 'Mozilla/5.0 (Linux; Android 7.0; SM-G892A Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/67.0.3396.87 Mobile Safari/537.36';
+
+    const items = await Promise.all(
+        feed.items.map(async (item) => {
+            // Fetch the AMP version
+            const url = item.link.replace(/^https:\/\/www\.scmp\.com/, 'https://amp.scmp.com');
+            const response = await got({
```

Comment:
need cache https://docs.rsshub.app/en/joinus/#submit-new-rss-rule-code-the-script-use-cache